### ISSUE TITLE
Hanging on Checking quarantine support step 

### ISSUE THREAD

**Issue by m-radzikowski** [Homebrew/homebrew-cask#126180]
### Verification

- [X] I understand that [if I ignore these instructions, my issue may be closed without review](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/closing_issues_without_review.md).
- [X] I have retried my command with `--force`.
- [X] I ran `brew update-reset && brew update` and retried my command.
- [X] I ran `brew doctor`, fixed as many issues as possible and retried my command.
- [X] I have checked the instructions for [reporting bugs](https://github.com/Homebrew/homebrew-cask#reporting-bugs).
- [X] I made doubly sure this is not a [checksum does not match](https://docs.brew.sh/Common-Issues#cask-checksum-does-not-match) error.

### Description of issue

When installing casks the homebrew hangs around 5-7 minutes on the "Checking quarantine support". Adding flag "--no-quarantine" does not help.

The issue https://github.com/Homebrew/brew/issues/9139 describes homebrew not honoring `--no-quarantine` and may be related, but it's fixed and merged, as well as the later fix for the `upgrade` command: https://github.com/Homebrew/brew/pull/10398

### Command that failed

brew install alacritty -vd --no-quarantine

### Output of command with `--verbose --debug`

```shell
Running `brew update --auto-update`...
==> Auto-updated Homebrew!
Updated 2 taps (homebrew/core and homebrew/cask).

You have 108 outdated formulae and 6 outdated casks installed.
You can upgrade them with brew upgrade
or list them with brew outdated.

==> Cask::Installer#install
==> Printing caveats
==> Cask::Installer#fetch
==> Downloading https://github.com/alacritty/alacritty/releases/download/v0.10.1/Alacritty-v0.10.1.dmg
/usr/local/Homebrew/Library/Homebrew/shims/shared/curl --disable --cookie /dev/null --globoff --show-error --user-agent Homebrew/3.5.2-115-gcc61192\ \(Macintosh\;\ Intel\ Mac\ OS\ X\ 12.4\)\ curl/7.79.1 --header Accept-Language:\ en --retry 3 --location --silent --head --request GET https://github.com/alacritty/alacritty/releases/download/v0.10.1/Alacritty-v0.10.1.dmg
Already downloaded: /Users/mradzikowski/Library/Caches/Homebrew/downloads/1e0cee0bc1c09b97b4b652ce9c43ac58d39f8b84e17667483ef1fc376cc366be--Alacritty-v0.10.1.dmg
==> Checking quarantine support
/usr/bin/xattr -h
/usr/bin/swift -target x86_64-apple-macosx12 /usr/local/Homebrew/Library/Homebrew/cask/utils/quarantine.swift
==> Unknown support status
==> Verifying checksum for cask 'alacritty'
hdiutil imageinfo -format /Users/mradzikowski/Library/Caches/Homebrew/downloads/1e0cee0bc1c09b97b4b652ce9c43ac58d39f8b84e17667483ef1fc376cc366be--Alacritty-v0.10.1.dmg
==> Installing Cask alacritty
Warning: macOS's Gatekeeper has been disabled for this Cask
==> Cask::Installer#stage
==> Extracting primary container
==> Using container class UnpackStrategy::Dmg for /Users/mradzikowski/Library/Caches/Homebrew/downloads/1e0cee0bc1c09b97b4b652ce9c43ac58d39f8b84e17667483ef1fc376cc366be--Alacritty-v0.10.1.dmg
hdiutil attach -plist -nobrowse -readonly -mountrandom /private/tmp/d20220621-23025-58cmzz /Users/mradzikowski/Library/Caches/Homebrew/downloads/1e0cee0bc1c09b97b4b652ce9c43ac58d39f8b84e17667483ef1fc376cc366be--Alacritty-v0.10.1.dmg
find . -print0
mkbom -s -i /private/tmp/20220621-23025-1qqsxa.list -- /private/tmp/20220621-23025-1mgi2e1.bom
ditto --bom /private/tmp/20220621-23025-1mgi2e1.bom -- /private/tmp/d20220621-23025-58cmzz/dmg.dvowT9 /private/tmp/d20220621-23025-2xolj7
diskutil info -plist /private/tmp/d20220621-23025-58cmzz/dmg.dvowT9
diskutil eject /private/tmp/d20220621-23025-58cmzz/dmg.dvowT9
cp -pR /private/tmp/d20220621-23025-2xolj7/Alacritty.app/. /usr/local/Caskroom/alacritty/0.10.1/Alacritty.app
chmod -Rf +w /private/tmp/d20220621-23025-2xolj7
==> Creating metadata directory: /usr/local/Caskroom/alacritty/.metadata/0.10.1/20220621104204.112
==> Creating metadata subdirectory: /usr/local/Caskroom/alacritty/.metadata/0.10.1/20220621104204.112/Casks
==> Installing artifacts
==> 10 artifacts defined
[I had to remove a long part here as it exceeded github issue text length limits]
==> Installing artifact of class Cask::Artifact::App
==> Moving App 'Alacritty.app' to '/Applications/Alacritty.app'
==> Installing artifact of class Cask::Artifact::Binary
==> Linking Binary 'alacritty' to '/usr/local/bin/alacritty'
/bin/ln -h -f -s -- /Applications/Alacritty.app/Contents/MacOS/alacritty /usr/local/bin/alacritty
==> Installing artifact of class Cask::Artifact::Binary
==> Linking Binary '_alacritty' to '/usr/local/share/zsh/site-functions/_alacritty'
/bin/ln -h -f -s -- /Applications/Alacritty.app/Contents/Resources/completions/_alacritty /usr/local/share/zsh/site-functions/_alacritty
==> Installing artifact of class Cask::Artifact::Binary
==> Linking Binary 'alacritty.bash' to '/usr/local/etc/bash_completion.d/alacritty'
/bin/ln -h -f -s -- /Applications/Alacritty.app/Contents/Resources/completions/alacritty.bash /usr/local/etc/bash_completion.d/alacritty
==> Adding com.apple.metadata:kMDItemAlternateNames metadata
/usr/bin/xattr -p com.apple.metadata:kMDItemAlternateNames /Applications/Alacritty.app/Contents/Resources/completions/alacritty.bash
==> Existing metadata is:
/bin/chmod -- u\+rw /Applications/Alacritty.app/Contents/Resources/completions/alacritty.bash /Applications/Alacritty.app/Contents/Resources/completions/alacritty.bash
/usr/bin/xattr -w com.apple.metadata:kMDItemAlternateNames \(\"alacritty\"\) /Applications/Alacritty.app/Contents/Resources/completions/alacritty.bash
==> Installing artifact of class Cask::Artifact::Binary
==> Linking Binary 'alacritty.fish' to '/usr/local/share/fish/vendor_completions.d/alacritty.fish'
/bin/ln -h -f -s -- /Applications/Alacritty.app/Contents/Resources/completions/alacritty.fish /usr/local/share/fish/vendor_completions.d/alacritty.fish
==> Installing artifact of class Cask::Artifact::Binary
==> Linking Binary 'alacritty-direct' to '/Users/mradzikowski/.terminfo/61/alacritty-direct'
/bin/ln -h -f -s -- /Applications/Alacritty.app/Contents/Resources/61/alacritty-direct /Users/mradzikowski/.terminfo/61/alacritty-direct
==> Installing artifact of class Cask::Artifact::Binary
==> Linking Binary 'alacritty' to '/Users/mradzikowski/.terminfo/61/alacritty'
/bin/ln -h -f -s -- /Applications/Alacritty.app/Contents/Resources/61/alacritty /Users/mradzikowski/.terminfo/61/alacritty
==> Installing artifact of class Cask::Artifact::Manpage
==> Linking Manpage 'alacritty.1.gz' to '/usr/local/share/man/man1/alacritty.1.gz'
/bin/ln -h -f -s -- /Applications/Alacritty.app/Contents/Resources/alacritty.1.gz /usr/local/share/man/man1/alacritty.1.gz
==> Installing artifact of class Cask::Artifact::Manpage
==> Linking Manpage 'alacritty-msg.1.gz' to '/usr/local/share/man/man1/alacritty-msg.1.gz'
/bin/ln -h -f -s -- /Applications/Alacritty.app/Contents/Resources/alacritty-msg.1.gz /usr/local/share/man/man1/alacritty-msg.1.gz
/usr/local/Homebrew/Library/Homebrew/shims/shared/git --version
🍺  alacritty was successfully installed!
```

### Output of `brew doctor --verbose`

```shell
==> Cask Environment Variables:
BUNDLE_PATH
CHRUBY_VERSION
GEM_HOME
GEM_PATH
HOMEBREW_CASK_OPTS
LC_ALL
PATH
RBENV_VERSION
RUBYLIB
RUBYOPT
RUBYPATH
SHELL
==> $LOAD_PATHS
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/warning-1.2.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/tapioca-0.7.2/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/yard-sorbet-0.6.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/yard-0.9.28/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/spoom-1.1.11/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/thor-1.2.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/sorbet-static-and-runtime-0.5.10098/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/sorbet-runtime-stub-0.2.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/sorbet-0.5.10098/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/sorbet-static-0.5.10098-universal-darwin-15/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/simplecov-cobertura-2.1.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/simplecov-0.21.2/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/simplecov_json_formatter-0.1.4/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/simplecov-html-0.12.3/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/ruby-macho-3.0.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rubocop-sorbet-0.6.10/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rubocop-rspec-2.11.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rubocop-rails-2.15.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rubocop-performance-1.14.2/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rubocop-1.30.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/unicode-display_width-2.1.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/ruby-progressbar-1.11.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rubocop-ast-1.18.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rspec_junit_formatter-0.5.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rspec-wait-0.0.9/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rspec-sorbet-1.8.3/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rspec-retry-0.6.2/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rspec-its-1.3.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rspec-github-2.3.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rspec-3.11.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rspec-mocks-3.11.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rspec-expectations-3.11.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rspec-core-3.11.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rspec-support-3.11.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/ronn-0.7.3/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rexml-3.2.5/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rdiscount-2.2.0.2/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/extensions/x86_64-darwin-15/2.6.0-static/rdiscount-2.2.0.2
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rbi-0.0.14/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/unparser-0.6.4/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rack-2.2.3.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/pry-0.14.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/plist-3.6.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/patchelf-1.3.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/parlour-8.0.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/sorbet-runtime-0.5.10098/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rainbow-3.1.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/parser-3.1.2.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/parallel_tests-3.11.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/parallel-1.22.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/mustache-1.1.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/method_source-1.0.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/mechanize-2.8.5/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/webrobots-0.1.2/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/webrick-1.7.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/rubyntlm-0.6.3/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/nokogiri-1.13.6-x86_64-darwin/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/racc-1.6.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/extensions/x86_64-darwin-15/2.6.0-static/racc-1.6.0
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/mini_portile2-2.8.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/net-http-persistent-4.0.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/net-http-digest_auth-1.4.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/mime-types-3.4.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/mime-types-data-3.2022.0105/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/json_schemer-0.2.21/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/uri_template-0.7.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/http-cookie-1.0.5/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/hpricot-0.8.6/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/extensions/x86_64-darwin-15/2.6.0-static/hpricot-0.8.6
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/hana-1.3.7/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/elftools-1.1.3/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/ecma-re-validator-0.4.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/regexp_parser-2.5.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/domain_name-0.5.20190701/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/unf-0.1.4/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/unf_ext-0.0.8.2/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/extensions/x86_64-darwin-15/2.6.0-static/unf_ext-0.0.8.2
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/docile-1.4.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/diff-lcs-1.5.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/did_you_mean-1.6.1/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/connection_pool-2.2.5/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/commander-4.6.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/highline-2.0.3/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/coderay-1.1.3/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/byebug-11.1.3/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/extensions/x86_64-darwin-15/2.6.0-static/byebug-11.1.3
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/bootsnap-1.12.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/extensions/x86_64-darwin-15/2.6.0-static/bootsnap-1.12.0
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/msgpack-1.5.2/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/extensions/x86_64-darwin-15/2.6.0-static/msgpack-1.5.2
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/bindata-2.4.10/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/ast-2.4.2/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/addressable-2.8.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/public_suffix-4.0.7/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/activesupport-6.1.6/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/zeitwerk-2.6.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/tzinfo-2.0.4/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/minitest-5.16.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/i18n-1.10.0/lib
/usr/local/Homebrew/Library/Homebrew/vendor/bundle/bundler/../ruby/2.6.0/gems/concurrent-ruby-1.1.10/lib/concurrent-ruby
/Library/Ruby/Gems/2.6.0/gems/did_you_mean-1.3.0/lib
/Library/Ruby/Site/2.6.0
/Library/Ruby/Site/2.6.0/x86_64-darwin21
/Library/Ruby/Site/2.6.0/universal-darwin21
/Library/Ruby/Site
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/vendor_ruby/2.6.0
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/vendor_ruby/2.6.0/x86_64-darwin21
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/vendor_ruby/2.6.0/universal-darwin21
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/vendor_ruby
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/x86_64-darwin21
/System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/lib/ruby/2.6.0/universal-darwin21
/usr/local/Homebrew/Library/Homebrew
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Unknown support status
==> Homebrew Version
3.5.2-115-gcc61192
==> macOS
12.4
==> SIP
Enabled
==> Homebrew Cask Staging Location
/usr/local/Caskroom
==> Homebrew Cask Taps:
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask (4011 casks)
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask-versions (220 casks)

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  ilmbase

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /Users/mradzikowski/.pyenv/shims/python3.9-config
  /Users/mradzikowski/.pyenv/shims/python-config
  /Users/mradzikowski/.pyenv/shims/python3-config
  /Users/mradzikowski/.pyenv/shims/pre-commit-validate-config
  /Users/mradzikowski/.pyenv/shims/python3.8-config

Warning: libiconv files detected at a system prefix other than /usr.
Homebrew doesn't provide a libiconv formula, and expects to link against
the system version in /usr. libiconv in other prefixes can cause
compile or link failure, especially if compiled with improper
architectures. macOS itself never installs anything to /usr/local so
it was either installed by a user or some other third party software.

tl;dr: delete these files:
  /usr/local/include/iconv.h
  /usr/local/include/iconv.h

Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause formulae that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  six
```

### Output of `brew tap`

```shell
gpanders/tap
homebrew/autoupdate
homebrew/bundle
homebrew/cask
homebrew/cask-versions
homebrew/core
```

**Comment by m-radzikowski** [comment]
`brew config`:

```
HOMEBREW_VERSION: 3.5.2-115-gcc61192
ORIGIN: https://github.com/Homebrew/brew
HEAD: cc6119261c25f2bca9c838fbcd24da47783cf85d
Last commit: 14 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 32e57bb3d15e95c86f405a0d0bb0fe9e076a7341
Core tap last commit: 2 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: ["--no-quarantine"]
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.VObgfjoHWv/org.macosforge.xquartz:0
HOMEBREW_MAKE_JOBS: 12
Homebrew Ruby: 2.6.8 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: dodeca-core 64-bit kabylake
Clang: 13.1.6 build 1316
Git: 2.34.1 => /usr/local/bin/git
Curl: 7.79.1 => /usr/bin/curl
macOS: 12.4-x86_64
CLT: 13.4.0.0.1.1651278267
Xcode: N/A
```

`brew doctor` (takes 8 minutes to complete, I don't know if that's normal or not):

```
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Unknown support status

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  ilmbase

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /Users/mradzikowski/.pyenv/shims/python3.9-config
  /Users/mradzikowski/.pyenv/shims/python-config
  /Users/mradzikowski/.pyenv/shims/python3-config
  /Users/mradzikowski/.pyenv/shims/pre-commit-validate-config
  /Users/mradzikowski/.pyenv/shims/python3.8-config

Warning: libiconv files detected at a system prefix other than /usr.
Homebrew doesn't provide a libiconv formula, and expects to link against
the system version in /usr. libiconv in other prefixes can cause
compile or link failure, especially if compiled with improper
architectures. macOS itself never installs anything to /usr/local so
it was either installed by a user or some other third party software.

tl;dr: delete these files:
  /usr/local/include/iconv.h
  /usr/local/include/iconv.h

Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause formulae that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  six
```

**Comment by carlocab** [comment]
> `brew doctor` (takes 8 minutes to complete, I don't know if that's normal or not):

That's definitely not normal. What about `brew doctor --debug`? That might take even longer; sorry. You may need to dump the output into a Gist to avoid the comment character limit. You can probably pipe the command into `pbcopy` so it goes straight to your clipboard.

**Comment by Bo98** [comment]
> ```
> <unknown>:0: error: module 'Darwin' is defined in both '/var/folders/hp/7l6ry4wd2_z444r11_c2_9c80000gn/C/clang/ModuleCache/2E8ZT5BS1DDBG/Darwin-CTM3NN7PD06L.pcm' and '/var/folders/hp/7l6ry4wd2_z444r11_c2_9c80000gn/C/clang/ModuleCache/2E8ZT5BS1DDBG/Darwin-16REU5I1P75TF.pcm'
> ```

This looks like your ModuleCache is broken and Swift can be very slow without a module cache. Try delete that folder `/var/folders/hp/7l6ry4wd2_z444r11_c2_9c80000gn/C/clang/ModuleCache/2E8ZT5BS1DDBG`

**Comment by m-radzikowski** [comment]
Funny thing - the directory `/var/folders/hp/7l6ry4wd2_z444r11_c2_9c80000gn/C/clang/ModuleCache/2E8ZT5BS1DDBG` is recreated when I (or Homebrew) run the `/usr/bin/swift -target x86_64-apple-macosx12 /usr/local/Homebrew/Library/Homebrew/cask/utils/quarantine.swift`, and it starts throwing errors right away. In other words, the `quarantine.swift` generates doubled cache files.

For other solutions/workarounds, is it expected that the `quarantine.swift` is run even when `--no-quarantine` flag is passed?

**Comment by Bo98** [comment]
> the directory `/var/folders/hp/7l6ry4wd2_z444r11_c2_9c80000gn/C/clang/ModuleCache/2E8ZT5BS1DDBG` is recreated

This is normal.

> it starts throwing errors right away

This is not normal and sugests something is perhaps corrupt in the SDK installation. Under the latest CLT version, only `Darwin-16REU5I1P75TF.pcm` should exist.

What does `ls -l /Library/Developer/CommandLineTools/SDKs` output?

Frew things to try:

* Try clear a level higher, so `/var/folders/hp/7l6ry4wd2_z444r11_c2_9c80000gn/C/clang/ModuleCache`
* Does reinstalling the CLT work? That involves doing `rm -rf /Library/Developer/CommandLineTools`, downloading the latest Command Line Tools for Xcode from developer.apple.com and installing it again (`xcode-select --install` also works be aware it's a bit buggy and doesn't always pick the latest so you may need to update it immediately after installing). You may also need to delete the module cache again when you do this as well.
